sire/intro.md

```diff
@@ -247,9 +231,33 @@ listy
 
 `TRUE` is represented as the nat `1`, while `FALSE` is the nat `0`.
 
+The `=?=` operation here is an assertion. It is often used for "unit testing". It takes two arguments and crashes if they are not equal, otherwise it evaluates without issue (it technically it takes a third argument, a continuation - the remainder of the program).
```

Comment:
I'm sorry if i've tricked you! 😭 But `=?=` doesn't take a third argument anymore than `=` does. The rest of the program is gobbled up by the Sire interpreter, which considers itself completely done with `=?=` after it has consumed its two arguments and checked that they evaluate to the same PLAN.

This situation might seem similar but is not the same as with `trk`, which literally does take two arguments on a PLAN level. `=?=` and `=` don't even exist on the PLAN level.

```suggestion
The `=?=` operation here is an assertion. It is often used for "unit testing". It takes two arguments and crashes if they are not equal, otherwise it evaluates without issue.
```

Reply:
As I _added_ that line about a third argument I thought "am I only writing this because I just explained this with `trk` or is this actually true?" and I intended to look at the source but forgot. 
:pray: thanks!